index.html

```diff
@@ -7,15 +7,78 @@
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
-    <!-- Add a link to your CSS file here (use the line above to guide you) -->
+    <!-- Add a link to your CSS file here (use the line above to guide you) -->su
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+
+
 <body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+<nav class = "navigation">
+  <img class = "logo" src="http://thinkful-ed.github.io/karma-clone/img/karma-logo.svg" alt = "Logo image" width = "26"/>
+  <ul class = "links">
+         <li><b>Meet Karma</b></li>
+         <li class ="style"> How it Works </li>
+         <li class ="style"> Store </li>
+         <li class ="style"> Blog </li>
+         <li class ="style"> Help </li>
+         <li class ="style"> Login </li>
+   </ul>  
+</nav>
+  
+  
+<section class = "introduction">
+  <div class = "introduction-text">
+      <h1>Introducing Karma</h1>
+      <h3>Bring Wifi with you</h3>
+      <button> Learn More </button>
+         
+  </div>
+</section>
+  
+  
+  
+  <section class = "Information-container">
+    <div class = "Information-text">
+      <h2> Everyone needs a little Karma</h2>
+    </div>
+         
+       
+           <ul class= "devices">  
+           <li>
+             <img width = "127" height = "127" src = "http://thinkful-ed.github.io/karma-clone/img/icon-devices.svg" alt = "internet for all devices.image"/>
+              <h3>Internet for all devices</h3>
+             </li>
+        
+             <li>
+             <img width = "127" height = "127" src = "http://thinkful-ed.github.io/karma-clone/img/icon-coffee.svg" alt = "Boost your productivity.image"/>
```

Comment:
please indent nested elements so that it's more readable, like this:
```
<li>
      <img width = "127" height = "127" src = "http://thinkful-ed.github.io/karma-clone/img/icon-coffee.svg" alt = "Boost your productivity.image"/>
<li>
```